igvc_gazebo/nodes/control/main.cpp

```diff
@@ -12,12 +12,9 @@ double speed_measured_left = 0.0;
 double speed_measured_right = 0.0;
 double speed_last_error_left = 0.0;
 double speed_last_error_right = 0.0;
-double speed_P_left = -20.0;
-double speed_P_right = -20.0;
-double speed_D_left = 1;
-double speed_D_right = 1;
+double effort_right, effort_left;
 
-constexpr double wheel_radius = 0.3429;
+double speed_P_left, speed_P_right, speed_D_left, speed_D_right, wheel_radius;
```

Comment:
These PID values should match those in motor_controller.launch as they're for setting the proper motor velocity